.github/workflows/build_docs.yml

```diff
@@ -0,0 +1,42 @@
+# Please keep in sync with /environment.yml
+name: Build Docs
+
+on: workflow_dispatch
+
+jobs:
+  build-docs:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: set git user
+      run: |
+        git config --global user.email "metomi@metoffice.gov.uk"
```

Comment:
I don't think this e-mail address is monitored any more even if it exists. Rich may have an idea for a more suitable option.

Reply:
This is the same as on PyPI, and Richard is now on holiday - sorry we left this so long! Perhaps we should make a ticket to cover both this and the PyPI email instead?

Reply:
Done, see #130